### DISCUSSION TITLE
Clarification on Supported Element Types for .case and .aedtplt File Formats

### DISCUSSION THREAD

**Discussion by chamanth-vct** [ansys/pyaedt#5320] [Q&A]
Hi,

In one of the projects I am currently working on, I need to export field results into a specific file format. From the documentation, I noticed that there are two options: `.case` and `.aedtplt` file types. At present, I am using the `.case` format.

However, during a discussion, I received feedback from the Ansys team stating that the `.case` file format only supports limited element types and does not support higher-order elements ([link to the comment](https://github.com/ansys/pyaedt/discussions/5182#discussioncomment-10691764)).

### Request for Information
To proceed with our development and plan the automation code effectively, I would like the following details:

- List of element types supported by the `.case` file format.
- List of element types not supported by the `.case` format, especially regarding higher-order elements.
- List of element types supported by the `.aedtplt` format.
- If file format support varies by application or solver type (e.g., HFSS, Maxwell), please specify the differences.

This information would greatly assist us in selecting the correct file type and optimizing our automation efforts.

Thank you

**Comment by Samuelopez-ansys** [comment]
Hi @chamanth-vct ,

These requests are not part of PyAEDT, this corresponds to Ansys product, please submit a request in the Ansys customer portal.

Regards,
Samuel
